term/patterns/outlined-media-and-text.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Title: Outlined Media and Text
+ * Slug: term/outlined-media-text
+ * Categories: media
+ * Block Types: core/media
```

Comment:
We use this when the pattern contains just that specific block type, I don't think it's the case here